modules/system-info/reporters/user.php

```diff
@@ -15,6 +15,10 @@
  */
```

Comment:
Did It affects the usage schema?

Reply:
Yes but i think it will be synced ( merged ) if it comes first then usage will trigger about and else too.